js/ironx.js

```diff
@@ -0,0 +1,655 @@
+            'baseVolume': this.safeFloat (volume, 'base'),
+            'quoteVolume': this.safeFloat (volume, 'quote'),
+            'info': ticker,
+        };
+    }
+
+    async fetchTrades (symbol, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        this.omit (params, 'page');
+        const request = {
+            'symbol': this.marketId (symbol),
+            'page': 1,
+            'itemsPerPage': limit || 10,
+        };
+        if (since !== undefined) {
+            request['startDate'] = since;
+        }
+        const response = await this.publicGetTradesAll (this.extend (request, params));
+        const trades = this.safeValue (response, 'result', []);
+        // return trades;
+        return this.parseTrades (trades, market, since, limit);
+    }
+
+    async fetchMyTrades (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        let market = undefined;
+        if (symbol) {
+            market = this.market (symbol);
+            symbol = this.marketId (symbol);
+        } else {
+            symbol = 'any';
+        }
+        const page = this.safeInteger (params, 'page', 1);
+        this.omit (params, 'page');
+        const request = {
+            'symbol': symbol,
+            'startDate': since,
+            'endDate': this.milliseconds (),
+            'page': page,
+            'itemsPerPage': limit,
+        };
+        const response = await this.privateGetUserTrades (this.extend (request, params));
+        const trades = this.safeValue (this.safeValue (response, 'result'), 'trades', []);
+        // return trades;
+        return this.parseTrades (trades, market, since, limit);
+    }
+
+    parseTrade (trade, market = undefined) {
+        const datetime = this.safeString (trade, 'created');
+        const price = this.safeFloat (trade, 'price');
+        const amount = this.safeFloat (trade, 'quantity');
+        const cost = price * amount; // total cost (including fees), `price * amount`
+        return {
+            'info': trade,
+            'id': this.safeString (trade, 'tradeId'),
+            'order': undefined,
+            'timestamp': this.parse8601 (datetime),
+            'datetime': datetime,
+            'symbol': this.safeString (trade, 'symbol', this.safeString (trade, 'pair')),
+            'type': undefined,
+            'side': undefined,
+            'takerOrMaker': undefined,
+            'price': price,
+            'amount': amount,
+            'cost': cost,
+            'fee': {
+                'cost': this.safeFloat (trade, 'fee'),
+                'currency': undefined,
+                'rate': undefined,
+            },
+        };
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        // const market = this.market (symbol);
```

Comment:
If you are leaving this here, write a comment to support that.

Reply:
Removed